Mathlib/Lean/Meta/Tactic/CleanupExcept.lean

```diff
@@ -0,0 +1,77 @@
+/-
+Copyright (c) 2023 Kyle Miller. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller
+-/
+import Lean.Meta.Tactic.Cleanup
+
+/-! # Configurable cleanup meta tactic
+
+This module defines `Lean.MVarId.cleanupExcept` as a configurable version of `Lean.MVarId.cleanup`.
```

Comment:
I've submitted a PR to change core so that `cleanup` has the same options as `cleanupExcept`: https://github.com/leanprover/lean4/pull/2919